src/main/java/seedu/address/model/parcel/DeliveryDate.java

```diff
@@ -16,6 +18,8 @@
     public static final String DATE_VALIDATION_REGEX =
             "^(0[1-9]|[12][0-9]|3[01])[- /.](0[1-9]|1[012])[- /.](19|20)\\d\\d$";
```

Comment:
What does this regex do? Oo

Reply:
It checks for the following:
It first splits the arguments by "/" "-" "." or " "
if the first argument starts with a 0, the following number must be from 1 to 9, otherwise, if the first argument starts with 1 or 2, the following number must be from 0 to 9, otherwise, if the first argument starts with a 3, the following number must be either 0 or 1.

if the second argument starts with a zero, the following number must be from 1 to 9, otherwise, if the second argument starts with a 1, the following number must be either 0,1 or 2. 

if the third argument can only start with 19 or 20, followed by 2 more digits which do not matter. 

But there's an error in the regex so I'll do up another PR for it.

Reply:
does it check whether the date is a valid date?

Reply:
For now it doesn't check, i'll make a PR to make fix it.